examples/apps/presence-tracker/tests/events.test.ts

```diff
@@ -0,0 +1,135 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * The test cases below ensure that type can be successfully imported from all three packages — tree, core-interfaces, and fluid-framework and validate that the imports work as expected.
+ * The plan is to remove types {@link @fluidframework/tree#Listenable}, {@link @fluidframework/tree#IsListener}, {@link @fluidframework/tree#Listeners} and {@link @fluidframework/tree#Off}
+ * from `@fluidframework/tree` in Fluid Framework 3.0 and instead import them from `fluid-framework` or `@fluidframework/core-interfaces`,
+ */
+
+import { strict as assert } from "node:assert";
+
+import { createEmitter } from "@fluid-internal/client-utils";
+import type { Listenable as Listenable_Interfaces } from "@fluidframework/core-interfaces";
+import type { Listenable as Listenable_Tree } from "@fluidframework/tree";
```

Comment:
Since these tests are specifically testing the interactions with tree, I think it might be best to group them with the other tree related import tests which were dumped in a different place, `experimental/framework/tree-react-api/src/test/importer.spec.ts`

Maybe if we eventually end up with enough such tests they can get a proper package for themselves someday.

Reply:
I made https://github.com/microsoft/FluidFramework/pull/23450 to give a proper place for this